packages/bitcore-lib/lib/transaction/input/taproot.js

```diff
@@ -103,4 +107,17 @@ TaprootInput.prototype.addSignature = function(transaction, signature) {
 };
 
 
+// TODO verify that this is the correct MAX size.
```

Comment:
Is this TODO TODONE?

Reply:
No. This works for the base case of just a normal wallet address, but if you stuff a tapscript into the p2tr, I doubt this is the max size. I will address this when I implement multisig taproot wallets.